src/Components/FeedbackPage/FeedbackForm/FeedbackForm.jsx

```diff
@@ -0,0 +1,204 @@
+import { useEffect, useState } from 'react';
+import { saveData } from '../../../DataService/localDataService';
+import { ErrorMessage } from '../../index';
+
+import styles from './FeedbackForm.module.scss';
+import { BiErrorCircle } from 'react-icons/bi';
+
+function FeedbackForm() {
+  const initialValues = {
+    username: '',
+    email: '',
+    comments: '',
+    newsletter: true,
+    feature: '',
+    feedbackType: ''
+  };
+  const [formData, setFormData] = useState(initialValues);
+  const [formErrors, setFormErrors] = useState({});
+  const [isSubmit, setIsSubmit] = useState(false);
+
+  function handleChange(event) {
+    const { name, value, type, checked } = event.target;
+    setFormData((prevFormData) => {
+      return {
+        ...prevFormData,
+        [name]: type === 'checkbox' ? checked : value
+      };
+    });
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    setFormErrors(validate(formData));
+    setIsSubmit(true);
```

Comment:
why not just save to local storage when `validate(formData)` returns empty object? in this case you wouldn't need isSubmit state at all

Reply:
I agree. Removed isSubmit state.